3-min-demo.mdx

```diff
@@ -27,19 +27,25 @@ You can [try it right now](https://app.octomind.dev/setup/url?utm_source=docs&ut
 
 ## 2. We will discover your user flows
 
-Full disclosure - we auto-discover a **sign-in** user flow and do **reachability** checks for all links on your page, but we'll add more manually.
-Or you can try out our brand new [free prompting feature](/discovery) beta release!
+At the first set-up, you can select if we should auto-discover:
 
-<Frame caption="Test user credentials input to test the sign-in flow, screenshot 09/2023">
+- a **log-in** user flow
+- a **change language** flow
```

Comment:
in the app we never talk about `flows` only about tests. It'd be great if our docs reflect that as well